### ISSUE TITLE
SuggestionsPopupWindow - IndexOutOfBoundsException: setSpan (9 ... 18) ends beyond length 9

### ISSUE THREAD

**Issue by david-allison-1** [ankidroid/Anki-Android#6170]
This happens to one user a lot, although with different numerical inputs: https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/reports-browser/user/2df3f390-8483-42e3-84c1-59900d3017e9 going back to 2.8.4

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/119332e7-2bc9-466c-9545-c59382f579a7

```
java.lang.IndexOutOfBoundsException: setSpan (9 ... 18) ends beyond length 9
at android.text.SpannableStringBuilder.checkRange(SpannableStringBuilder.java:1325)
at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:684)
at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:676)
at android.widget.Editor$SuggestionsPopupWindow.updateSuggestions(Editor.java:4430)
at android.widget.Editor$SuggestionsPopupWindow.show(Editor.java:4264)
at android.widget.Editor.replace(Editor.java:573)
at android.widget.-$$Lambda$DZXn7FbDDFyBvNjI-iG9_hfa7kw.run(Unknown Source:2)
at android.os.Handler.handleCallback(Handler.java:883)
at android.os.Handler.dispatchMessage(Handler.java:100)
at android.os.Looper.loop(Looper.java:237)
at android.app.ActivityThread.main(ActivityThread.java:7860)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1075)
```

**Comment by david-allison-1** [comment]
User doesn't have logging, and we don't seem to store the current activity.

**Comment by david-allison-1** [comment]
Fixed in #7352
